### PR TITLE
Adding some language about grammars to the README

### PR DESCRIPTION
**Pull request by arfon**
/ cc @Alhadis

### CODE REVIEW

**Comment by Alhadis** [comment]
> which are included in this project as a set of submodules. [This README](https://github.com/github/linguist/blob/master/vendor/README.md) lists all of these grammars together with their location.

Could probably be condensed to:

> which are bundled with this project as a list of submodules, [which may be found here](https://github.com/github/linguist/blob/master/vendor/README.md).

Otherwise, looks good. 👍

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ The Language stats bar displays languages percentages for the files in the repos
 0. If the files are being misclassified, search for [open issues][issues] to see if anyone else has already reported the issue. Any information you can add, especially links to public repositories, is helpful.
 0. If there are no reported issues of this misclassification, [open an issue][new-issue] and include a link to the repository or a sample of the code that is being misclassified.
 
+### There's a problem with the syntax highlighting of a file
+
+Linguist detects the language of a file but the actual syntax-highlighting is powered by a set of language grammars which are included in this project as a set of submodules. [This README](https://github.com/github/linguist/blob/master/vendor/README.md) lists all of these grammars together with their location. 
+
+If you experience an issue with the syntax-highlighting on GitHub **please report the issue on the upstream grammar repository and not here.** Grammars are updated everytime we build the Linguist gem and so upstream bug fixes will be automatically incorporated as they are fixed.
```
Comment (Alhadis) [inline README.md]: > If you experience an issue with the syntax-highlighting on GitHub

Needs a comma.

> please report the issue ~~**on**~~ the

\* "To the" sounds better.

> the upstream grammar repository

Needs a comma.

> **~~and~~** not here

Can drop the word "and".

> Grammars are updated everytime

It's actually spelt "every time", with a space... although it wouldn't surprise me if this gets misused so often, it winds up a canonical part of the English language... :( 

> we build the Linguist gem, and **~~so~~** upstream bug fixes **~~will be~~** are automatically incorporated as they are fixed.

**Comment by Alhadis** [comment]
Sorry for the grammar nazi-ing... but look on the bright side: years of picking on people's grammar paid off in the form of syntax highlighting, hahaha.

**Comment by arfon** [comment]
⚡️ thanks for the review @Alhadis

**Comment by Alhadis** [comment]
Always happy to be a pedantic prick. ^_^;

File: README.md (hunk 1 above)
Comment (pchaigno) [inline README.md]: I'm gonna ask you to proofread all my articles :heart:

**Pull request merged**